.github/workflows/multiarch.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Multi-arch builds
```

Comment:
```suggestion
name: Multi-arch Builds
```

Reply:
We can go ahead and merge this, fix the name in a follow-up commit as was done in other repos.